packages/Discretization/test/tstExodusAsciiGenerator.cpp

```diff
@@ -0,0 +1,107 @@
+/****************************************************************************
+ * Copyright (c) 2012-2017 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ ****************************************************************************/
+
+#include "DTK_ConfigDefs.hpp"
+#include "DTK_Types.h"
+
+#include "PointCloudProblemGenerator/PointCloudProblemGenerator.hpp"
+#include "PointCloudProblemGenerator/ExodusAsciiGenerator.hpp"
+
+#include <Kokkos_View.hpp>
+
+#include <Teuchos_UnitTestHarness.hpp>
+#include <Teuchos_CommHelpers.hpp>
+#include <Teuchos_RCP.hpp>
+#include <Teuchos_DefaultComm.hpp>
+
+#include <memory>
+#include <string>
+
+//---------------------------------------------------------------------------//
+// Partition the grids one-to-one
+TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( ExodusAsciiGenerator, one_to_one, Node )
+{
+    // Type aliases.
+    using DeviceType = typename Node::device_type;
+
+    // Get the communicator.
+    auto comm = Teuchos::DefaultComm<int>::getComm();
+
+    // Create a problem generator.
+    std::string src_coord_file = "fine_sphere_node_coords.dat";
```

Comment:
Where is that file? If it is in DTK-Data, we need to copy it when we compile the test.

Reply:
yes needs to get added to dtk data